workflows/rnaseq/rnaseq-variant-calling.wdl

```diff
@@ -0,0 +1,132 @@
+# SPDX-License-Identifier: MIT
+# Copyright St. Jude Children's Research Hospital
+version 1.1
+
+import "../../tools/picard.wdl"
+import "../../tools/gatk4.wdl" as gatk
+
+workflow rnaseq_variant_calling {
+    meta {
+        description: "Call variants from RNA-Seq data. Produces a VCF file of variants. Based on GATK RNA-Seq short variant calling best practices pipeline."
+        outputs: {
+            recalibrated_bam: "BAM that has undergone recalibration of base quality scores"
+            recalibrated_bam_index: "Index file for recalibrated BAM file"
+            merged_vcf: "VCF of raw RNA-Seq variant calls"
```

Comment:
Does the user care it's been merged? I wouldn't care. I supplied one bam, I expect one VCF. I would rename this, but IDK what. Plain `vcf` is a little terse, and we've gone on record saying output names should be "descriptive". `unfiltered_vcf`? Does that make it sound like an intermediate file? `raw_vcf`?

Reply:
Another thought on this. Isn't this VCF strictly a subset of the information present in the filtered VCF? Is there any use for it? If a user doesn't care about the annotations/filters applied, they can always be ignored (is "filtered" the right word? Variants which fail get annotated, not removed, right?).

Reply:
According to the tool documentation, it should be possible to remove the variants. However, I don't see a command line flag that actually enables that feature. Also, I definitely don't expose such a parameter. So you're correct, we could just output the filtered VCF. It's a little more cumbersome to ignore the filters in analysis, but it is possible.

Reply:
> it should be possible to remove the variants. However, I don't see a command line flag that actually enables that feature.

🙄 

> we could just output the filtered VCF

My thinking is that it's simpler to only have one output VCF. No user is going to ask us "which should I use?" Worst case someone down the line complains (with a valid use case for the "unfiltered" VCF) and we add this line back 🤷